ansible/roles/baselayout/vars/main.yml

```diff
@@ -57,6 +57,25 @@ packages: {
     'bash,cmake,gcc-c++,gcc6-c++,tar,unzip,git,make,sudo',
   ],
 
+  # @jgorzins - this works, but @rvagg says doing it as a concatenated string
```

Comment:
@ThePrez I'll make this change, once I get ansible to run (having ssh auth problems ATM)

Reply:
Thank you!

Reply:
@sam-github If you make a change can you add back `m4-gnu`?
https://github.com/sam-github/build/blob/08b957ef686da692f3d440059a2274075c4fb18d/ansible/roles/baselayout/vars/main.yml#L59

(and maybe `cmake` too)